test_runner/fixtures/pageserver/http.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, res: requests.Response):
         super().__init__(res.json()["msg"], res.status_code)
 
 
+class TimelineCreate409(PageserverApiException):
```

Comment:
Maybe `TimelineCreateConflict` or `TimelineCreationConflict` instead? But there is already a 406 (Not Acceptable) variant.. From the PR desc, I did not understand how can this fix anything :)

Reply:
the https://github.com/neondatabase/neon/blob/7ea8c85e471489e1ef7d3b9b9e1e6fcac37014b1/test_runner/regress/test_remote_storage.py#L741 

expects PageserverApiException and not Exception